tests/e2e/support/objects/app-files/resource/actions.ts

```diff
@@ -479,35 +481,69 @@ export const restoreResourceVersion = async (args: restoreResourceVersionArgs) =
 }
 
 /**/
-
 export interface deleteResourceArgs {
   page: Page
-  resource: string
+  resource?: string
```

Comment:
needed now? Didn't see its use

Reply:
created another args type to make this more strict